### PR TITLE
add single thread page content and style

### PR DESCRIPTION
**Pull request by Didisimmons**
Add single thread page content and style

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -27,24 +27,23 @@ <h2 class="header-text">Find your tribe, lose the language barrier </h2>
...
-    <div class="topthread-container">
-      <div class="row">
-          {% for thread in threads %}
-            <div class="col-12 text-center">
-              <h4 class="single-thread"><a href="{{ url_for('thread', threadID=thread._id) }}" class="view-more-thread">{{ thread.subject }}</a></h4>
-            </div>
-          {% endfor %}
-          <p class="text-center view-more-threads "><a href="{{url_for('thread')}}" class="link-color view-more-thread" >Start New Thread</a></p>
+      <h3 class="text-center header-content">Latest Discussions</h3>
+      <p class="summary-text text-center">We are here to create a community were people from all over the world can come and
```
Comment (kera-cudmore) [inline templates/index.html:31]: line 31 and 32 - you need to update were to where 🙂

**Review by kera-cudmore — CHANGES_REQUESTED** [review]
Just a little one - left comment on index page = line 31 and 32 need were changed to where 🙂

**Review by andrewdempsey2018 — APPROVED** [review]
LGTM 🤜

**Comment by Didisimmons** [comment]
Fix indentation on style sheet

**Review by andrewdempsey2018 — APPROVED** [review]
LGTM 🤜

**Pull request merged**